core/common/src/main/java/alluxio/security/authentication/plain/SaslClientHandlerPlain.java

```diff
@@ -92,4 +96,15 @@ public ChannelAuthenticationScheme getClientScheme() {
   public SaslClient getSaslClient() {
     return mSaslClient;
   }
+
+  @Override
+  public void close() {
+    if (mSaslClient != null) {
+      try {
+        mSaslClient.dispose();
+      } catch (SaslException exc) {
+        LOG.debug("Failed to close SaslClient.", exc);
```

Comment:
Is it a problem if we fail to close the client or server?

Reply:
Not really. But I don't expect that to happen often, that's why the debug trace.